openslides_backend/models/motion_change_recommendation.py

```diff
@@ -0,0 +1,35 @@
+from ..shared.patterns import Collection
+from . import fields
+from .base import Model
+
+
+class MotionChangeRecommendation(Model):
+    """
+    Model for motion change_recommendation.
+
+    Reverse fields:
+    - motion_ids
+    """
+
+    collection = Collection("motion_change_recommendation")
+    verbose_name = "motion_change_recommendation"
+
+    id = fields.IdField(description="The id of this motion change recommendation.")
+    title = fields.RequiredCharField(
+        description="The title of this motion change recommendation."
+    )
+    type = fields.PositiveIntegerField("The type of this motion change recommendation.")
```

Comment:
Add comment:
Replacement (0), Insertion (1), Deletion (2), Other (3)
add max=3 (see #101, you can use any jsonschema constraints on the fields)

Reply:
Add comment and max=3.